### PR TITLE
Shuff4

### PR DESCRIPTION
**Pull request by shuff4**

### CODE REVIEW

File: community/people/SH/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+Layout: Person
```
Comment (gonuke) [inline community/people/SH/index.md:2]: I think these all need to be lower case `layout`, etc...  The data is not being processed properly with the capitalized version of these tags

**Review by gonuke — CHANGES_REQUESTED** [review]
This looks mostly correct.  I think the data labels need to be lower case to be processed correctly.

File: community/people/SH/index.md (hunk 1 above)
Comment (shuff4) [reply community/people/SH/index.md]: okay, made sure I changed to lower case

File: community/people/sh/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+Layout: Person
```
Comment (gonuke) [inline community/people/sh/index.md:2]: It is these labels, e.g. `layout` that need to be lower case

**Review by gonuke — CHANGES_REQUESTED** [review]
Thanks for the updates, but it's the labels in the headers that need to be lowercase

File: community/people/sh/index.md (hunk 2 above)
Comment (shuff4) [reply community/people/sh/index.md]: Okay, I think that should do it :], hopefully. I didn't realize it the first time you mentioned it.

File: community/people/sh/index.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+layout: person
+title: ShaiAnthony Huff
+image: IMG-0456.jpg
+role: Nuclear Engineering Undergrad
+email: 
+  - huff.shaianthony8@gmail.com 
+room: virtual
```
Comment (gonuke) [inline community/people/sh/index.md:8]: This all looks great now when I test it.  Given the way the room shows up, you may just want to leave this entry out of your description entirely.

**Review by gonuke — CHANGES_REQUESTED** [review]
It looks like you have the mechanics of this figured out, and we're one small step away from merging this pull request.

File: community/people/sh/index.md (hunk 3 above)
Comment (shuff4) [reply community/people/sh/index.md]: okay, I have removed it.

**Comment by gonuke** [comment]
LGTM (Looks Good To Me) - thanks @shuff4

**Review by gonuke — APPROVED** [review]

**Pull request merged**